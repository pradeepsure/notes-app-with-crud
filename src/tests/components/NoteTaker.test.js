import NoteTaker from '../../components/NoteTaker';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

let wrapper, notes, addNote;
beforeEach(() => {    
    notes= [{
        "noteTitle": "Hello",
        "noteDescription": "Take Mediscine Reminder",
        "id": 1
      }]
      addNote = jest.fn();
      wrapper = shallow(
        <NoteTaker addNote={addNote} />
    );
});


test('Render NoteTaker component correctly', () => {        
    expect(wrapper).toMatchSnapshot();
});