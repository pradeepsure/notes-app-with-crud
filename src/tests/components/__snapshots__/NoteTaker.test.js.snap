// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NoteTaker component correctly 1`] = `
<NoteTaker
  addNote={[MockFunction]}
  classes={
    Object {
      "absolute": "NoteTaker-absolute-2",
      "fab": "NoteTaker-fab-1",
    }
  }
/>
`;
